### DISCUSSION TITLE
Can Widevine be installed?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [waydroid/waydroid#1058] [Q&A] [answered]
https://github.com/waydroid/waydroid/discussions?discussions_q=is%3Aopen+widevine returns 0 results.

**Comment by YogSottot** [comment, marked as answer]
> returns 0 results.

Because a link to a script that can install this is in the documentation

https://github.com/casualsnek/waydroid_script#integrate-widevine-drm-l3

**Reply by roke-julian-lockhart** [reply]
> https://github.com/waydroid/waydroid/discussions/1058#discussioncomment-8210133

@YogSottot, why is it not included in the default image?

**Reply by roke-julian-lockhart** [reply]
> https://github.com/waydroid/waydroid/discussions/1058#discussioncomment-8210196

https://github.com/waydroid/waydroid/discussions/1058#discussioncomment-8210969

**Comment by electrikjesus** [comment]
It's proprietary so we can't include it by default.
